core/src/kura.rs

```diff
@@ -878,9 +879,10 @@ mod tests {
             .unwrap()
```

Comment:
Please update the documentation of the test

Reply:
Ok